.gitignore

```diff
@@ -41,7 +41,7 @@ coverage.xml
 .pytest_cache/
 
 # pyenv
-.python_version
+.python-version
```

Comment:
Why are we changing this back after https://github.com/stripe/stripe-python/commit/469b3d6cac85ad639d308c06241f93b2b017228c#diff-bc37d034bad564583790a46f19d807abfe519c5671395fd494d8cce506c42947 ?

Reply:
Oh, didn't even notice this. `python-version` is correct and 
https://github.com/stripe/stripe-python/commit/469b3d6cac85ad639d308c06241f93b2b017228c#diff-bc37d034bad564583790a46f19d807abfe519c5671395fd494d8cce506c42947 was mistaken.